app/components/text-field/component.js

```diff
@@ -9,10 +9,12 @@ export default Ember.Component.extend({
 
     textFieldValueObserver: Ember.observer('textFieldValue', function() {
         this.set('widget.parameters.value.value', this.get('textFieldValue'));
+        this.get('widget.parameters.value').save();
     }),
 
     didReceiveAttrs() {
         this.set('description', this.attrs.description);
+        this.set('textFieldValue', this.get('widget.parameters.value.value'));
```

Comment:
Need to get it because for some reason it doesn't always bind right away. It may have something to do with the parameters computed in the widget model as it is undefined for a short period.